src/requests_utilities.py

```diff
@@ -92,186 +101,88 @@ def get(
             logger.info("Response has empty body (Content-Length: 0)")
             return None
 
-        self.response_json = self.response_api.json()
+        if "application/json" not in self.response_api.headers.get(
+            "Content-Type", ""
+        ):
```

Comment:
Can we skip it if "application/json" is empty? Sending empty "Content-Type" - looks weird